src/routes/view-content/library-resource-menu/LibraryResourceMenu.svelte

```diff
@@ -118,7 +118,8 @@
         currentFullscreenResourceGrouping = resourceGrouping;
     }
 
-    async function prepareResources(resources: ResourceContentInfo[]) {
+    async function prepareResources(resources: ResourceContentInfo[], isShowing: boolean) {
+        if (!isShowing) return;
```

Comment:
Could we add a check if it's already initialized like `!isShowing || resourceGroupings` so it doesn't trigger every time the tab is clicked but only the first time? If we did that we'd also want to wrap the `<LibraryResourceMenu ... />` on the page in a `{#key $selectedBibleSection}` so that when the Bible section changes it will render a new instance of this component.